drivers/gpu/drm/tests/drm_framebuffer_test.c

```diff
@@ -381,6 +381,27 @@ static void drm_test_framebuffer_create(struct kunit *test)
 	KUNIT_EXPECT_EQ(test, params->buffer_created, buffer_created);
 }
 
+static void drm_test_framebuffer_modifiers_not_supported(struct kunit *test)
+{
+	/* Valid cmd with DRM_MODE_FB_MODIFIERS set */
+	struct drm_mode_fb_cmd2 cmd = {
+		.width = MAX_WIDTH, .height = MAX_HEIGHT,
+		.pixel_format = DRM_FORMAT_ABGR8888, .handles = { 1, 0, 0 },
+		.offsets = { UINT_MAX / 2, 0, 0 }, .pitches = { 4 * MAX_WIDTH, 0, 0 },
+		.flags = DRM_MODE_FB_MODIFIERS,
+	};
+	struct drm_device *mock = test->priv;
+	int buffer_created = 0;
+
+	mock->dev_private = &buffer_created;
+	mock->mode_config.fb_modifiers_not_supported = 1;
+
+	drm_internal_framebuffer_create(mock, &cmd, NULL);
+	KUNIT_EXPECT_EQ(test, 0, buffer_created);
+
+	mock->mode_config.fb_modifiers_not_supported = 0;
+}
+
```

Comment:
Wouldn't be better to add this test as a parameterized test?

Reply:
I think so, but to parametrize it I should move the drm `drm_device` mock in the `drm_framebuffer_tests` struct and so include its fields in all the existent cases, that would implies repeat exactly the same fields in all these tests but the one that I would be adding, which differs to the others by the `.mode_config.fb_modifiers_not_supported` field of the `drm_device` mock.

Does it worth to make the test in the way I described or there is a better way that I hadn't think yet?

Reply:
Yeah, it seems that this is the best strategy. Could you just remove the comment on line 386?

Reply:
> Yeah, it seems that this is the best strategy. Could you just remove the comment on line 386?

Did that in the last forced-push, thanks!